packages/contracts/deploy/verification/99_conclude/10_verify-contracts.ts

```diff
@@ -27,7 +27,7 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   }
 };
 export default func;
-func.tags = ['New', 'Verify'];
+func.tags = ['New', 'Verify', 'VerifyContracts'];
```

Comment:
1- Is this should run automatically or we trigger this manually according to the checklist?
2- what is the difference between 'Verify' & 'VerifyContracts'

Reply:
1. This runs automatically. Verify proxies does not.
2. Verify refers to a collection of verification scripts, including some runtime checks for permissions and state. VerifyContracts refers just to this step.

Reply:
Resolving due to no further comments

Reply:
2. the meaning are too close to each other, and it is confusing at first glance, if we could fined a better naming, that would be better

3. noticed also that this file and etherscan.ts both has delay() repeated, so there is duplicated code, not related to ur work, but can be improved